internal/server/volume/server.go

```diff
@@ -0,0 +1,144 @@
+package volume
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/klog"
+
+	"github.com/kubernetes-csi/csi-proxy/client/apiversion"
+	"github.com/kubernetes-csi/csi-proxy/internal/server/volume/internal"
+)
+
+type Server struct {
+	hostAPI API
+}
+
+type API interface {
+	ListVolumesOnDisk(diskID string) (volumeIDs []string, err error)
+	// MountVolume mounts the volume at the requested global staging path
+	MountVolume(volumeID, path string) error
+	// DismountVolume gracefully dismounts a volume
+	DismountVolume(volumeID, path string) error
```

Comment:
`UnmountVolume` is better

Reply:
Although I agree with you on this, Dismount is the terminology used in powershell world. Hence keeping it as Dismount for similarity in this context.

Reply:
I am fine with either and we discussed this briefly. To @andyzhangx 's point, `unmount` is used everywhere in k8s codebase.